### PR TITLE
Fixes #13866: Invalid json in Rudder inventory hooks fails to log the problem 

### PR DESCRIPTION
**Pull request by peckpeck**
https://issues.rudder.io/issues/13866

### CODE REVIEW

File: rudder-agent/SOURCES/patches/fusioninventory/11726-Extend-Rudder-specific-inventory-with-clie.patch [hunk 1]
```diff
@@ -122,6 +122,11 @@ index b2fd04d..41e1dea 100644
 +                };
 +                if ($@) {
 +                    $logger->error("Script $script_file didn't return valid JSON entry, error is:$@") if $logger;
++                    my $filename = "/tmp/inventory-json-error-".time();
```
Comment (amousset) [inline rudder-agent/SOURCES/patches/fusioninventory/11726-Extend-Rudder-specific-inventory-with-clie.patch]: maybe `/var/rudder/tmp/`, or at least add rudder in the name and make it only readable by root, we don't know what these variables contains.

Comment (amousset) [reply rudder-agent/SOURCES/patches/fusioninventory/11726-Extend-Rudder-specific-inventory-with-clie.patch]: @peck ping

**Comment by amousset** [comment]
Needs a retarget to 5.0

**Comment by peckpeck** [comment]
PR updated with a new commit

**Comment by peckpeck** [comment]
PR replaced by https://github.com/Normation/rudder-packages/pull/2100

**Pull request closed without merging**